### ISSUE TITLE
Price Performance Analysis of Top 1000 Crypto Assets and Their Implications for Oxen

### ISSUE THREAD

**Issue by venezuela01** [loki-project/loki-improvement-protocols#57]
# Price Performance Analysis of Top 1000 Crypto Assets and Their Implications for Oxen

The Oxen token price has fallen from its all-time high of $2.35 on April 23, 2021, to $0.79, registering a loss of 96.62%.

This study extracts price information from the top 1000 and top 100 crypto assets listed on [CoinMarketCap](https://coinmarketcap.com/) and compares their current prices with their respective all-time highs.

Although analyzing market capitalization would offer a more accurate reflection of the value of crypto projects—especially inflationary tokens like Oxen—we opted for price data. Our marketcap data source from [TokenInsight](https://tokeninsight.com/) lacks a sufficiently long historical record to capture the peak in the last bull market. Therefore, we use token price as a proxy to measure value change.

## Percentile Table for Top 1000 Crypto Assets

| Percentile | Price Change Percentage |
|------------|----------|
| 10th       | -99.371  |
| 20th       | -98.660  |
| 30th       | -97.856  |
| 40th       | -96.800  |
| 50th       | -95.230  |
| 60th       | -93.970  |
| 70th       | -91.207  |
| 80th       | -84.996  |
| 90th       | -67.498  |

As the percentile table indicates, the majority of the top 1000 crypto assets have suffered substantial declines, with a median fall exceeding 95%. Oxen's decline of 96.62% aligns closely with the broader market.

## Scatter Chart Analysis by Market Cap

After breaking down the data by each crypto asset's current market capitalization, we plotted a scatter chart comparing market cap with the percentage of price change. The chart reveals that most crypto assets exhibit uniformly poor performance, irrespective of their market cap. In other words, leading projects only slightly outperform the tail end of the top 1000 in terms of price changes.

![top_1000_price_performance](https://github.com/oxen-io/oxen-improvement-proposals/assets/138533453/29e71445-0bf1-4a7d-856a-e7d6ac45ff46)

## Percentile Table for Top 100 Crypto Assets

| Percentile | Price Change Percentage |
|------------|----------|
| 10th       | -97.173  |
| 20th       | -95.952  |
| 30th       | -93.648  |
| 40th       | -92.136  |
| 50th       | -88.245  |
| 60th       | -84.228  |
| 70th       | -70.579  |
| 80th       | -61.396  |
| 90th       | -26.198  |

Half of the top 100 assets have fallen more than 88% from their all-time highs.

It's worth noting that Oxen's performance isn't positive; its market cap ranking has dropped from around 500th from its peak to approximately 1000th at the moment. However, this study provides quantitative data that enables balanced decision-making. The price drop in Oxen should not be exclusively attributed to its privacy features. Instead, our analysis shows that Oxen's price decline is in line with the macroeconomic trends affecting the majority of crypto assets.

**Comment by venezuela01** [comment]
[Top1000_CoinMarketCap_Sorted.csv](https://github.com/oxen-io/oxen-improvement-proposals/files/12505278/top1000_coinmarketcap_sorted.csv)

The data source was scraped from CoinMarketCap with minimal post-processing. It is attached for anyone interested in semi-independently validating the conclusions.

**Comment by venezuela01** [comment]
See also [ORC-8](https://github.com/oxen-io/oxen-improvement-proposals/issues/36)

**Comment by Paul1804** [comment]
> # Price Performance Analysis of Top 1000 Crypto Assets and Their Implications for Oxen
> The Oxen token price has fallen from its all-time high of $2.35 on April 23, 2021, to $0.79, registering a loss of 96.62%.
> 
> This study extracts price information from the top 1000 and top 100 crypto assets listed on [CoinMarketCap](https://coinmarketcap.com/) and compares their current prices with their respective all-time highs.
> 
> Although analyzing market capitalization would offer a more accurate reflection of the value of crypto projects—especially inflationary tokens like Oxen—we opted for price data. Our marketcap data source from [TokenInsight](https://tokeninsight.com/) lacks a sufficiently long historical record to capture the peak in the last bull market. Therefore, we use token price as a proxy to measure value change.
> 
> ## Percentile Table for Top 1000 Crypto Assets
> Percentile	Price Change Percentage
> 10th	-99.371
> 20th	-98.660
> 30th	-97.856
> 40th	-96.800
> 50th	-95.230
> 60th	-93.970
> 70th	-91.207
> 80th	-84.996
> 90th	-67.498
> As the percentile table indicates, the majority of the top 1000 crypto assets have suffered substantial declines, with a median fall exceeding 95%. Oxen's decline of 96.62% aligns closely with the broader market.
> 
> ## Scatter Chart Analysis by Market Cap
> After breaking down the data by each crypto asset's current market capitalization, we plotted a scatter chart comparing market cap with the percentage of price change. The chart reveals that most crypto assets exhibit uniformly poor performance, irrespective of their market cap. In other words, leading projects only slightly outperform the tail end of the top 1000 in terms of price changes.
> 
> ![top_1000_price_performance](https://user-images.githubusercontent.com/138533453/265245393-29e71445-0bf1-4a7d-856a-e7d6ac45ff46.png)
> 
> ## Percentile Table for Top 100 Crypto Assets
> Percentile	Price Change Percentage
> 10th	-97.173
> 20th	-95.952
> 30th	-93.648
> 40th	-92.136
> 50th	-88.245
> 60th	-84.228
> 70th	-70.579
> 80th	-61.396
> 90th	-26.198
> Half of the top 100 assets have fallen more than 88% from their all-time highs.
> 
> It's worth noting that Oxen's performance isn't positive; its market cap ranking has dropped from around 500th from its peak to approximately 1000th at the moment. However, this study provides quantitative data that enables balanced decision-making. The price drop in Oxen should not be exclusively attributed to its privacy features. Instead, our analysis shows that Oxen's price decline is in line with the macroeconomic trends affecting the majority of crypto assets.

You're correct. They should take their time and carefully consider the situation.  The optimal approach is to focus on enhancing their current product and implementing more assertive marketing strategies.
